.pre-commit-config.yaml

```diff
@@ -0,0 +1,28 @@
+repos:
+  - repo: https://github.com/pre-commit/pre-commit-hooks
+    rev: v4.6.0
+    hooks:
+      - id: check-yaml
+        args:
+          - "--unsafe"
+      - id: end-of-file-fixer
+      - id: trailing-whitespace
+  - repo: https://github.com/psf/black-pre-commit-mirror
```

Comment:
Consider switching to ruff for formatting and black. See all the branches on DM-48788 -- ruff is so much faster than black+isort and you are already running ruff.